bindings/experimental/distrdf/python/DistRDF/HeadNode.py

```diff
@@ -358,15 +358,18 @@ def __init__(self, backend: BaseBackend, npartitions: Optional[int], *args):
         super().__init__(backend, npartitions)
 
         self.defaultbranches = None
-        self.friendinfo = None
+        # Information about friend trees. It is not None only if the TTree
+        # constructor overload is used and if there is at least one friend.
```

Comment:
this comment will go out of date fast: "...only if the TTree constructor _or the RDatasetSpec constructor_"